silk-react-components/src/api/silkRestApi.js

```diff
@@ -0,0 +1,135 @@
+import superagent from '@eccenca/superagent';
+import Promise from 'bluebird';
```

Comment:
I know the file is only renamed more or less, so all the adds are copy&paste, but is it really intended to user superagent AND bluebird? Both do exactly the same in its core. Fun fact: In future superagent will be replaced by axios (the third big player for http requests)